jest.config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
       "setupTestFrameworkScriptFile": "<rootDir>/testHelper/setupTests.ts",
       "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
       "transform": {
-        "^.+\\.tsx?$": "ts-jest"
+        "^.+\\.tsx?$": "ts-jest",
+        "^.+\\.js?$": "ts-jest"
```

Comment:
Needed this for tests to run on components that have imports from `design-tokens`. Is there a better solution?

Reply:
I ended up changing the design-tokens JS exports to commonJS, but it still appears to be failing in CI :(